File: Baekjoon/Class2/22.10814 나이순 정렬.py
import sys
input = sys.stdin.readline

n = int(input())
people = []
for _ in range(n):
    age, name = input().rstrip().split(" ")
    people.append([int(age), name])

people.sort(key = lambda person: person[0])
print("\n".join([f"{person[0]} {person[1]}" for person in people]))

File: Baekjoon/Class2/22.10814 나이순 정렬.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim().split(/\s+/);

let cursor = 0;
const n = Number(input[cursor++]);
const people = [];
for(let _ = 0; _ < n; _++)
    people.push([Number(input[cursor++]), input[cursor++]]);

people.sort((p1, p2) => p1[0] - p2[0]);
console.log(people.map(person => `${person[0]} ${person[1]}`).join("\n"));